examples/Cargo.toml

```diff
@@ -70,7 +70,13 @@ opsram-2m = ["esp-hal/opsram-2m"]
 psram-2m  = ["esp-hal/psram-2m"]
 
 [profile.release]
-debug = true
+codegen-units    = 1
+debug            = 2
+debug-assertions = false
+incremental      = false
+opt-level        = 3
+lto = 'fat'
```

Comment:
Building examples with fat lto now, we may as well use the best settings available.